packages/client/src/storage.ts

```diff
@@ -127,10 +127,10 @@ export type Storage$ = <Type extends StorageItemInput["type"]>(
 const isOptionalArg = (lastArg: any) => {
   if (typeof lastArg !== "object") return false
 
-  return Object.keys(lastArg).every(
-    (k) =>
-      (k === "at" && typeof lastArg.at === "string") ||
-      (k === "signal" && lastArg.signal instanceof AbortSignal),
+  return Object.entries(lastArg).every(
+    ([k, v]) =>
+      (k === "at" && (v === undefined || typeof v === "string")) ||
+      (k === "signal" && (v === undefined || v instanceof AbortSignal)),
   )
 }
```

Comment:
GH dosn't allow me to make this suggestion, but since you are fixing this could you please also remove the bogus `if`? Like this:

```ts
const isOptionalArg = (lastArg: any) =>
  typeof lastArg === "object" &&
  Object.entries(lastArg).every(
    ([k, v]) =>
      (k === "at" && (v === undefined || typeof v === "string")) ||
      (k === "signal" && (v === undefined || v instanceof AbortSignal)),
  )
```

Reply:
Yep, have committed the change. Also, accidentally discovered another potential bug by changing `any` to `unknown` 😆.

Reply:
good catch! yep, `typeof null` => `object` 💯